src/background/providers/cloudflare.test.ts

```diff
@@ -40,8 +40,11 @@ test('setStoredTokens', () => {
     const provider = new CloudflareProvider(storage, { updateIcon, navigateUrl });
     const tokens = [new Token(), new Token()];
     provider['setStoredTokens'](tokens);
-    const storedTokens = JSON.parse(storage.store.get('tokens')!);
-    expect(storedTokens).toEqual(tokens.map((token) => token.toString()));
+    const tok = storage.store.get('tokens');
+    if (tok) {
```

Comment:
Could this be more explicit whether it's checking for `false`, `null`, or `undefined`?

Reply:
done